test/unit/logging/NotificationLoggerSpec.scala

```diff
@@ -31,19 +35,26 @@ class NotificationLoggerSpec extends UnitSpec with MockitoSugar {
   trait SetUp {
     val mockCdsLogger: CdsLogger = mock[CdsLogger]
     val logger = new NotificationLogger(mockCdsLogger)
+
+    implicit val mockHeaderCarrier: HeaderCarrier = mock[HeaderCarrier]
+    when(mockHeaderCarrier.headers).thenReturn(LoggingHeaders)
   }
 
   "NotificationsLogger" should {
     "debug(s: => String, headers: => SeqOfHeader)" in new SetUp {
-      logger.debug("msg", LoggingHeaders)
+      import uk.gov.hmrc.apinotificationpull.controllers.CustomHeaderNames.getHeadersFromHeaderCarrier
```

Comment:
signature of `debug(s: => String, headers: => SeqOfHeader)` has changed and should be reflected here if you want to keep the same format. Same applies to other tests below